docs/METRICS.md

```diff
@@ -225,6 +225,14 @@ These are events that an add-on user can encounter on a shot they own
 7. [x] Click download from tile: `web/download/myshots-tile`
 8. [x] Clear search with button: `web/clear-search/button`
 
+#### Settings (My Shots)
+
+1. [x] Click connect button `web/start-connect/settings`
```

Comment:
Is there an event when the login completes?  Seems like there should be.  It might have to be sent from the server instead of the web.  (There's other examples in the server of sending GA events.)